app/models/katello/authorization/content_view.rb

```diff
@@ -25,7 +25,7 @@ def promotable_or_removable?
     end
 
     def exportable?
```

Comment:
I don't think any one is calling this. Can ya confirm and remove if that's the case (i just `git grep exportable app` and couldnt find any  references.)